### ISSUE TITLE
Support RUMBLE download

### ISSUE THREAD

**Issue by TechWatchRadio** [ytdl-org/youtube-dl#30717]
## Checklist

- [X] I'm reporting a feature request
- [X] I've verified that I'm running youtube-dl version **2021.12.17**
- [ ] I've searched the bugtracker for similar feature requests including closed ones

## Description

Please work on Rumble download support — Thanks!

**Comment by dirkf** [comment]
Duplicate of #10785, which was partially implemented (for embed URLs), then left hanging.

yt-dlp has some further changes that support general Rumble URLs, and these can be back-ported. Some examples of current URLs needed by OP would be useful for testing.

**Comment by bill-mcgonigle** [comment]
I wound up here for this URL and can confirm that the -dlp module works.
https://rumble.com/vwxxi8-ukraine-on-fire.html

**Comment by dirkf** [comment]
Indeed, no situation is so bad that an Oliver Stone (exec producer) conspiracy film can't make it worse. Luckily we have the technology to demonstrate this, or will do when I or someone else gets around to the back-port.

**Comment by 89z** [comment]
https://github.com/ytdl-org/youtube-dl/issues/31036

**Comment by Aphexus** [comment]
[generic] v1nmqdh: Requesting header
[redirect] Following redirect to https://rumble.com/v1nmqdh-albert-einstein-talks-about-war-nationalism-democracy-and-the-usa.html
[generic] v1nmqdh-albert-einstein-talks-about-war-nationalism-democracy-and-the-usa: Requesting header
WARNING: Falling back on generic information extractor.
[generic] v1nmqdh-albert-einstein-talks-about-war-nationalism-democracy-and-the-usa: Downloading webpage
[generic] v1nmqdh-albert-einstein-talks-about-war-nationalism-democracy-and-the-usa: Extracting information
ERROR: Unsupported URL: https://rumble.com/v1nmqdh-albert-einstein-talks-about-war-nationalism-democracy-and-the-usa.html
